src/main/java/gnu/io/NativeResource.java

```diff
@@ -83,8 +83,7 @@ public synchronized void load(String libraryName) throws NativeResourceException
 				}
 				return;
 			} catch (Exception e){
-				e.printStackTrace();
-				throw new NativeResourceException("Unable to load native resource from given path.\n" + e.getLocalizedMessage());
+				throw new NativeResourceException("Unable to load native resource from given path" + e.getLocalizedMessage());
```

Comment:
I agree that the newline is undesirable, but we probably want _some_ whitespace separation:

```suggestion
				throw new NativeResourceException("Unable to load native resource from given path: " + e.getLocalizedMessage());
```

Reply:
Good catch!